### DISCUSSION TITLE
Detecting multiple cats

### DISCUSSION THREAD

**Discussion by suhlig** [blakeblackshear/frigate#10205] [Ask A Question] [answered]
My cat is getting harassed my the neighbor's cat, and I would like to detect when two cats are in the image and one of them is our cat.

I have the detection basically working for one cat or one person, but I cannot seem to be able to detect multiple cats. Should this work?

Is the detection of "our" cat even possible? Perhaps with a custom model?

Thanks, and keep up the good work.

**Comment by NickM-27** [comment, marked as answer]
what makes you think multiple cats can't be detected? where are you looking for this info?

and no, detection of something specific needs an image classification model not an object detection model

**Reply by suhlig** [reply]
> what makes you think multiple cats can't be detected? where are you looking for this info?

I do not remember seeing multiple objects of the same type (label?) mentioned in the documentation, but that might be a misunderstanding on my side.

> and no, detection of something specific needs an image classification model not an object detection model

Thanks for the clarification!

**Reply by NickM-27** [reply]
frigate tracks all objects that are detected, there is no limitation to one of every type

**Comment by xury77** [comment]
I have a very similar problem. When I have more than one tracked object in the frame at the same time, only one is detected. My tracked objects are: person, car, dog, cat.

**Reply by NickM-27** [reply]
are you sure only one is detected, have you looked at the debug view? Only having one event is different than only having one detected

**Comment by xury77** [comment]
However, I think that if it detected more than one object, the 'person count' entity should be greater than 1 and it never is.

**Reply by NickM-27** [reply]
without some specifics there isn't really any advice that can be offered. 
![Screen Shot 2024-03-05 at 11 40 59 AM](https://github.com/blakeblackshear/frigate/assets/14866235/c47ba6e2-fa85-4505-9301-b6b73ea59fa8)
![Screen Shot 2024-03-05 at 11 41 55 AM](https://github.com/blakeblackshear/frigate/assets/14866235/8121482f-6917-4459-8b28-7db6b00ff200)
